build.rs

```diff
@@ -41,6 +41,19 @@ mod musl_reference_tests {
         "rem_pio2.rs",
         "rem_pio2_large.rs",
         "rem_pio2f.rs",
+        "remquo.rs",
+        "remquof.rs",
```

Comment:
Something I should have done last time is add a comment here, but could a comment be added to these files indicating that they don't have public APIs and are only considered to be implementation details?

Reply:
Ok.

Reply:
These are supposed to be public APIs.